### ISSUE TITLE
2 tests fail on FreeBSD

### ISSUE THREAD

**Issue by yurivict** [thermohub/thermofun#37]
```
==================================================================================== test session starts =====================================================================================
platform freebsd13 -- Python 3.9.13, pytest-7.1.2, pluggy-1.0.0 -- /usr/local/bin/python3.9
cachedir: .pytest_cache
hypothesis profile 'default' -> database=DirectoryBasedExampleDatabase('/disk-samsung/freebsd-ports/science/thermofun/work/thermofun-0.4.0/.hypothesis/examples')
rootdir: /disk-samsung/freebsd-ports/science/thermofun/work/thermofun-0.4.0, configfile: pytest.ini, testpaths: ThermoFun, pytests
plugins: forked-1.4.0, rerunfailures-10.1, hypothesis-6.50.1, xdist-2.5.0, cov-2.9.0, typeguard-2.13.3, subtests-0.8.0
collected 19 items                                                                                                                                                                           

pytests/test_database.py::TestDatabase::test_add_json_reaction_substance PASSED                                                                                                        [  5%]
pytests/test_database.py::TestDatabase::test_append_data PASSED                                                                                                                        [ 10%]
pytests/test_database.py::TestDatabase::test_append_element PASSED                                                                                                                     [ 15%]
pytests/test_database.py::TestDatabase::test_formula_parser PASSED                                                                                                                     [ 21%]
pytests/test_database.py::TestDatabase::test_parsing_database PASSED                                                                                                                   [ 26%]
pytests/test_database.py::TestDatabase::test_parsing_reaction PASSED                                                                                                                   [ 31%]
pytests/test_database.py::TestDatabase::test_parsing_substance PASSED                                                                                                                  [ 36%]
pytests/test_thermoengine.py::TestThermoEngine::test_properties_reaction FAILED                                                                                                        [ 42%]
pytests/test_thermoengine.py::TestThermoEngine::test_properties_reaction_from_equation FAILED                                                                                          [ 47%]
pytests/test_thermoengine.py::TestThermoEngine::test_properties_substance PASSED                                                                                                       [ 52%]
pytests/test_thermoengine.py::TestThermoEngine::test_properties_substance_from_reaction PASSED                                                                                         [ 57%]
pytests/Reactions/test_logKfT.py::TestThermoEngine::test_properties_logKfT_reaction PASSED                                                                                             [ 63%]
pytests/Reactions/test_rbm.py::TestThermoEngine::test_properties_Ryzhenko_Bryzgalin_reaction PASSED                                                                                    [ 68%]
pytests/Substances/Gases/test_gas_PRSV.py::TestCpfT::test_properties_substance PASSED                                                                                                  [ 73%]
pytests/Substances/Solids/test_CpfT_phase_transition.py::TestCpfT::test_properties_substance PASSED                                                                                    [ 78%]
pytests/Substances/Solute/test_hkf_gems.py::TestHKFGems::test_3kbar_al_ion PASSED                                                                                                      [ 84%]
pytests/Substances/Solute/test_hkf_gems.py::TestHKFGems::test_3kbar_sio2 PASSED                                                                                                        [ 89%]
pytests/Substances/Solute/test_hkf_gems.py::TestHKFGems::test_psat_al_ion PASSED                                                                                                       [ 94%]
pytests/Substances/Solute/test_hkf_gems.py::TestHKFGems::test_psat_sio2 PASSED                                                                                                         [100%]

========================================================================================== FAILURES ==========================================================================================
_________________________________________________________________________ TestThermoEngine.test_properties_reaction __________________________________________________________________________

self = <test_thermoengine.TestThermoEngine testMethod=test_properties_reaction>

    def test_properties_reaction(self):
        assert self.engine2.thermoPropertiesReaction(298.15, 1e5, "Meionite-Ca").log_equilibrium_constant.val == pytest.approx(80.873916, 1e-5, 1e-14)
>       assert self.engine2.thermoPropertiesReaction(423.15, 0, "Meionite-Ca").log_equilibrium_constant.val == pytest.approx(28.243799, 1e-5, 1e-14)
E       assert 28.2299676214565 == 28.243799 ± 2.8e-04
E         comparison failed
E         Obtained: 28.2299676214565
E         Expected: 28.243799 ± 2.8e-04

pytests/test_thermoengine.py:32: AssertionError
__________________________________________________________________ TestThermoEngine.test_properties_reaction_from_equation ___________________________________________________________________

self = <test_thermoengine.TestThermoEngine testMethod=test_properties_reaction_from_equation>

    def test_properties_reaction_from_equation(self):
        assert self.engine.thermoPropertiesReaction(298.15, 1e5, "Cal = Ca+2 + CO3-2").log_equilibrium_constant.val == pytest.approx(-8.48014, 1e-5, 1e-14)
        assert self.engine.thermoPropertiesReaction(298.15, 1e5, "Al+3 + 4 H2O@ + 0Ca+2= 1Al(OH)4- + 4 \n H+").log_equilibrium_constant.val == pytest.approx(-22.3085, 1e-5, 1e-14)
>       assert self.engine.thermoPropertiesReaction(423.15, 0, "Cal = Ca+2 + CO3-2").log_equilibrium_constant.val == pytest.approx(-10.10169, 1e-5, 1e-14)
E       assert -10.10141561200007 == -10.10169 ± 1.0e-04
E         comparison failed
E         Obtained: -10.10141561200007
E         Expected: -10.10169 ± 1.0e-04

pytests/test_thermoengine.py:25: AssertionError
================================================================================ 2 failed, 17 passed in 2.69s ================================================================================
*** Error code 1
```

**Comment by gdmiron** [comment]
In any case the differences are insignificant from a geochemical perspective, probably some roundoff differences in Ubuntu vs freebsd, the expected values were computed in ubuntu.

**Comment by gdmiron** [comment]
fixed in #50
